.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,21 +1,27 @@
 ---
 name: Bug
-about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing list https://groups.google.com/forum/#!forum/janusgraph-users"
+about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing list https://lists.lfaidata.foundation/g/janusgraph-users/topics"
 labels: "kind/bug/possible"
 
 ---
-
-Please post all questions and issues on [janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)
+<!--
+Please post all questions and issues on [janusgraph-users](https://lists.lfaidata.foundation/g/janusgraph-users/topics)
 before opening a GitHub issue. Your questions will reach a wider audience there,
-and if we confirm that there is a bug, then you can open a new issue.
+and if we confirm that there is a bug, then you can open a new issue. You could also use GitHub Discussions or Gitter.
```

Comment:
I think it would be good to include direct links to GitHub Discussions and Gitter.
I.e.
`[GitHub Discussions](https://github.com/JanusGraph/janusgraph/discussions) or [Gitter](https://gitter.im/janusgraph/janusgraph)`